web-common/src/components/search/Search.svelte

```diff
@@ -34,7 +35,9 @@
     bind:this={ref}
     type="text"
     autocomplete="off"
-    class="bg-white border border-gray-200 focus:border-blue-400 outline-none rounded-sm block w-full pl-8 p-1"
+    class="bg-white border border-gray-200 {showBorderOnFocus
+      ? 'focus:border-blue-400'
+      : ''} outline-none rounded-sm block w-full pl-8 p-1"
```

Comment:
This approach isn't great for accessibility ([convo with ChatGPT](https://chat.openai.com/share/6464b832-880e-49d6-8db3-98c9ca0e0a2d))

If we don't like the blue outline, I'd suggest we ask Janet to consider another focus state or we can turn off `autofocus`

Reply:
I would not be in favour of turning off autofocus for UI concerns. Probably best to see how we can redesign search for cases where the search bars are focused on mount by default.